### ISSUE TITLE
Sending message to Gitter is broken

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.Recipe#246]
```
Sending message to Gitter...
Resolving assembly GitterSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
Exception while resolving assembly GitterSharp: Could not load file or assembly 'GitterSharp' or one of its dependencies. The system cannot find the file specified.
System.IO.FileNotFoundException: Could not load file or assembly 'GitterSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
File name: 'GitterSharp, Version=1.0.0.0, Culture=
neutral, PublicKeyToken=null'
   at Cake.Gitter.Chat.GitterChatApi.PostToChatApi(ICakeContext context, String message, GitterChatMessageSettings messageSettings)
   at Cake.Gitter.Chat.GitterChatApi.PostMessage(ICakeContext context, String message, GitterChatMessageSettings messageSettings)
   at Submission#0.SendMessageToGitterRoom()
WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
```

See https://ci.appveyor.com/project/cakecontrib/cake-issues/build/0.4.1.build.137#L850

**Comment by pascalberger** [comment]
Caused by https://github.com/cake-contrib/Cake.Gitter/issues/34

**Comment by gep13** [comment]
@pascalberger this has been fixed in 0.10.0 of Cake.Gitter.

**Comment by pascalberger** [comment]
@gep13 Thanks for taking care of this
